sec7-application.tex

```diff
@@ -1,33 +1,19 @@
 \section{Application of the Science Platform inside the Project}\label{application-of-the-science-platform-inside-the-project}
 
-The LSP is not only for use by scientist end users through the Data Access
-Centers.  The Platform will be used extensively within the LSST project as well
-for a number of purposes.
+The Science Platform is not only for use by scientist end users through the Data Access Centers but it also forms an integral part of the toolchain within the Project. A phased release of capabilities focuses on the needs of the primary target audience for each major version. This approach allows us to deliver features of the platform to project staff to assist them in contruction work while also providing an early user base whose feedback will allow us to refine our design. 
 
-\subsection{Developer Support}\label{developer-support}
+\subsection{Data Management Teams}\label{developer-support}
 
-Developers will use the LSP to visualize and analyze the results of code tests,
-including both periodic and ad hoc runs using both simulated and real data.
-The Integration environment LSP instance will be used to verify and validate
-new features and updates to the LSP code, deployment, services, and processes.
+The earliest releases of the Science Platform are aimed at Data Management teams. The capabilities supporting software development include interfactively troubleshooting code and analysing metrics through the Notebook Aspect. The capabilities supporting algorithmisc development include visualising the results of large processing runs of precursor and simulated data through the Portal Aspect. 
 
-\subsection{Commissioning}\label{commissioning}
+\subsection{Commissioning Team}\label{commissioning}
 
-The Commissioning Team will use the Commissioning Cluster instance of the LSP
-to rapidly visualize and analyze images taken for system verification and
-validation purposes; to analyze the results of experiments with hardware,
-firmware, and software configuration modifications; and to prototype quality
-control data products and checks to be incorporated into automated systems.
-The Commissioning Cluster instance has direct access to the live Engineering
-and Facilities Database, so it can be used to correlate image and image
-quality measurement behavior with trends in telemetry.
+Intermediary releases of the Science Platform are aimed at the commissioning team. The capabilities supporting commissioning include the interfaces needed to allow Notebooks to cross-correlate early ComCam data with telemetry captured in the Engineering Facilities Database and a deployment of the Science Platform on the Commissioning Cluster.
```

Comment:
The examples are still in there but I prefer to deal with it as system capabilities rather than who is doing what (it being an arch document rather than a conops doc)